README.md

```diff
@@ -43,9 +43,23 @@ Now that you have the mParticle Dart plugin, install mParticle on your native/we
 
 To install mParticle on an Android platform:
 
-1. Grab your mParticle key and secret from [your workspace's dashboard](https://app.mparticle.com/apps) and construct an `MParticleOptions` object.
+1. Add the following dependencies to your app's `build.gradle`:
 
-2. Call `start` from the `onCreate` method of your app's `Application` class. It's crucial that the SDK be started here for proper session management. If you don't already have an `Application` class, create it and then specify its fully-qualified name in the `<application>` tag of your app's `AndroidManifest.xml`.
+```groovy
+dependencies {
+    implementation 'com.mparticle:android-core:5+'
+
+    // Required for gathering Android Advertising ID (see below)
```

Comment:
```suggestion
    // Required for gathering Android Advertising ID
```
Seems the 'see below' note is not applicable here.

Reply:
I'll update this in another PR.  It looks like this commit leaked over. This PR no longer has this.